.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -1,12 +1,45 @@
 ---
 name: Feature Request
-about: You want to suggest a new feature for Atmosphère.
-
+about: Suggest a new feature for Atmosphère.
+#assignees:
 ---
 
 ## Feature Request
-
+[ If any section does not apply, use nil. ]  
+[ Lines between [ ] (square brackets) should be removed before posting. ]
 ### What feature are you suggesting?
+#### Overview:
+- [ Include the basic, high-level concepts for this feature here. ]
 
-### Why would this feature be useful?
+#### Smaller Details: 
+- [ These may include specific methods of implementation etc. ]
+
+#### Nature of Request:  
+[ Remove all that do not apply to your request. ]
+- Addition
+  - [ Ex: Addition of certain original features or features from other community projects. ]
+  - [ If you are suggesting porting features or including features from other projects, include what license they are distributed under and what, if any libraries those project use. ]
+- Change
+- Removal
+  - [Ex: Removal of certain features or implementation due to a specific issue/bug or because of low quality code, etc.]
 
+[ Required: Reference the already existing [feature request. ](https://github.com/Atmosphere-NX/Atmosphere/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+label%3A%22features%2Ffeature-request%22) before you make your own request. ]  
+[ Failure to do so means your request can and likely will be closed with no warning. ]
```

Comment:
Again, I don't think this is actually a reasonable thing for us to do, even if the user didn't bother to search for open feature requests.
```suggestion
```

Reply:
It should fall upon the user to consult the open feature requests. It is your job as to developer to write code rather than deal with duplicate issues.

Reply:
It really should, but if users don't check for open feature requests, I believe that it's our tasks as *maintainers* to keep the open issue list clean and provide justification for why we're closing issues. This phrasing is uninviting and implies that we're going to be closing issues without any clarification or justification.

Reply:
I will definitely rephrase it to something that would inspire confidence in potential contributors. Cheers.

Reply:
Addressed and rephrased.